docs/code-search/working/saved_searches.mdx

```diff
@@ -1,50 +1,38 @@
-# Saved Search
+# Saved Searches
 
 <p className="subtitle">This page provides docs about how Saved Searches work with Sourcegraph.</p>
 
-Saved Search lets you save and describe search queries so you can easily find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories.
+Saved Searches lets you easily save and describe search queries to find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories.
 
-If you want Sourcegraph to monitor a search query and send notifications when there are new results, use [code monitoring](/code_monitoring/).
+Saved Searches functionality is available to both Free and Enterprise Code Search users. To access or create new Saved Searches in the Sourcegraph web app, click the **Tools > Saved Searches** in the top navigation bar.
 
-## Creating saved searches
-
-A saved search consists of a description and a query, both of which you can define and edit.
+![saved-searhces](https://storage.googleapis.com/sourcegraph-assets/Docs/saved-searches-0824.png)
 
-Saved searches can be associated with a user or an org:
+## Creating saved searches
 
-- User saved searches are only visible to (and editable by) the user that created them.
-- Org saved searches are visible to (and editable by) all members of the org.
+To create a new saved search:
 
-### User saved searches
+- Go to the Saved Searches section and click the **New saved search** button
+- Fill out the description field and define the syntax for the search query
```

Comment:
"Define the syntax" sounds wrong. I'd say it's "enter the search query". Define the syntax sounds like you are defining a new way of specifying search queries